### DISCUSSION TITLE
How to update shipping price during checkout with a custom price?

### DISCUSSION THREAD

**Discussion by madmonkeyworks** [saleor/saleor#14020] [Q&A] [answered]
Hallo, 

We offer our clients several timeslots a day which they can select and have their order delivered in that particular time. Each timeslot has its own shipping price. Instead of creating all the timeslots as individual payment methods, we are looking for a way to update the shipping price during the checkout. Is that possible?

**Comment by timuric** [comment, marked as answer]
Yes, you can do it via webhook https://docs.saleor.io/docs/3.x/developer/extending/webhooks/synchronous-events/shipping
